docs/examples/production/rabbitmq.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: production
+spec:
+  replicas: 3
+  resources:
+    requests:
+      cpu: 2
+      memory: 8Gi
+    limits:
+      cpu: 2
+      memory: 8Gi
+  rabbitmq:
+    additionalConfig: |
+      cluster_partition_handling = ignore
+      vm_memory_high_watermark_paging_ratio = 0.99
+      disk_free_limit.relative = 1.0
+  persistence:
+    storageClassName: ssd
+    storage: "128Gi"
```

Comment:
500Gi gives us the optimal performance for this workload: https://github.com/rabbitmq/infrastructure/blob/6c0d4b5eccdc7dc3c7256ac0c553b711d4004210/commercial-edition/docker/rmq/k8s/yaml/perftest